<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoonInfoTabWidget</class>
 <widget class="QWidget" name="MoonInfoTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>77</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="obs_date_label">
       <property name="text">
        <string>Observation Date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="obs_date_edit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="obs_time_label">
       <property name="text">
        <string>Observation Time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="obs_time_edit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="location_label">
       <property name="text">
        <string>Location</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="location_edit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
